Request/AbstractRequestBodyParamConverter.php

```diff
@@ -151,25 +157,33 @@ protected function execute(Request $request, ParamConverter $configuration)
     }
 
     /**
-     * @return DeserializationContext
+     * @return ContextInterface
+     */
+    protected function getContextAdapter()
+    {
+        return $this->container->get('fos_rest.context.adapter.chain_context_adapter');
```

Comment:
That's a good question ... I will create a call action.

And why isn't the serializer injected ?